modules/virt-viewing-downward-metrics-cli.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virt/monitoring/virt-exposing-downward-metrics.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-viewing-downward-metrics-cli_{context}"]
+= Viewing downward metrics by using the command line
+
+You can view downward metrics by entering a command from inside a guest virtual machine (VM).
+
+.Procedure
+
+* Run the following command:
```

Comment:
Nit - there are technically two commands users are instructed to use in this step:
```suggestion
* Run the following commands:
```